.golangci.yml

```diff
@@ -0,0 +1,38 @@
+---
+run:
+  concurrency: 6
+  deadline: 5m
+linters:
+  disable-all: true
+  enable:
+    - deadcode
+    - depguard
+    - dupl
+    - gochecknoinits
+    - goconst
+    - gofmt
+    - ineffassign
+    - interfacer
+    - misspell
+    - structcheck
+    - typecheck
+    - varcheck
+    # - errcheck
+    # - gochecknoglobals
+    # - gocritic
+    # - gocyclo
+    # - goimports
+    # - golint
+    # - gosec
+    # - govet
+    # - lll
+    # - maligned
+    # - megacheck
+    # - nakedret
+    # - prealloc
+    # - scopelint
+    # - unconvert
+    # - unparam
```

Comment:
The latest version of golangci-lint (1.17.1) contains a new checker called `bodyclose`. We should add them to this list too to be complete (either enabled or disabled).